compiler/src/iree/compiler/Codegen/Common/TransformExtensions/CommonExtensions.cpp

```diff
@@ -378,20 +378,21 @@ static void addUnrollVectorsGpuMmaSyncPatterns(RewritePatternSet &patterns) {
                     .setUnrollTraversalOrderFn(unrollOrder));
 }
 
-static std::optional<SmallVector<int64_t>> getGPUTensorCoreNativeWmmaVectorSize(
-    Operation *op) {
-  return getWmmaNativeVectorSize(op);
-}
-
-static void addUnrollVectorsGpuWmmaPatterns(RewritePatternSet &patterns) {
+static void addUnrollVectorsGpuWmmaPatterns(
+    RewritePatternSet &patterns, SmallVector<int64_t, 3> &mnkShape,
+    llvm::DenseMap<Operation *, WMMAFragType> &typeMap) {
```

Comment:
I see, applying directly to the op list makes sense but it would probably be tricky to get the ordering right (e.g. transfer_reads/ops on the inputs get the shape from the slice coming from the unrolled contract).